### ISSUE TITLE
Remove node-sass dependency from 1.x

### ISSUE THREAD

**Issue by ogonkov** [thoughtbot/neat#643]
### Issue Summary

Having dependency from `node-sass` cause errors with `node@8.x`

### Steps to Reproduce

1. Install `bourbon-neat@1.8.0` on `node@8.9.4`

`npm` failed to install `node-sass` that in `dependencies` of `bourbon-neat@1.8.0`

### Technical details:

- Neat Version: 1.8.0
- Build Tool or Environment: node@8.9.4, npm@5.6.0
- Browser/OS: Centos 6.9

**Comment by ogonkov** [comment]
`node-sass` seems doesn't required for `bourbon-neat` to install, can we remove it from dependencies for 1.x as it was done for 2.x?

**Comment by tysongach** [comment]
Other people have had similar trouble, but we worried that if we remove a dependency within a minor release that we might break people's projects inadvertently. So instead we published a branch here on GitHub ([`neat-1.8.0-node-sass`](https://github.com/thoughtbot/neat/tree/neat-1.8.0-node-sass)) that has node-sass removed.

You should be able to consume that specific branch like this:

```json
"dependencies": {
  "neat": "thoughtbot/neat#neat-1.8.0-node-sass"
}
```
